src/test-geogram-install2.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include <geogram/mesh/mesh.h>
+
+void toto_mesh()
+{
+    using namespace GEO;
+
+    initialize();
+
+    Mesh M;
+
+    vec3 point{};
+    M.vertices.create_vertex( point.data() );
+
+    std::cout << MeshCellDescriptors::tet_descriptor.nb_facets << std::endl;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of "::std::cout" by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS106)